### DISCUSSION TITLE
Big differences between Tle2Orbit and stk/GNI1B

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#287] [Q&A]
I use the groops tle2orbit program and stk soft to predict orbit. There are some big differences between 
Figure 1 shows the differences between GROOPS and STK, while Figure 2 illustrates the differences between GROOPS and GNI1B. From the figures, it can be observed that the differences in the Y and Z directions calculated by GROOPS are quite large, while the difference in the X direction is very small. Additionally, the difference between GROOPS and the official GNI1B orbit is in the range of several hundred meters. Could you please advise on which configuration issues might be causing these discrepancies? Thank you!
![253c14197b5b23ef92e6ccf266ba289](https://github.com/user-attachments/assets/09178095-411c-4dcf-b81e-5e2c04ea32dc)
![ee5230596efa1fa360a2fe86c4b9046](https://github.com/user-attachments/assets/bfba520e-b189-487a-bc04-eb776a106624)
The settings for STK are as follows, 
![9bdb03fd591ff372ff4733af4cd1032](https://github.com/user-attachments/assets/58db8fe7-516b-421b-8bc8-78fd6658e117)
and the settings for GROOPS are as follows. 
![ee5230596efa1fa360a2fe86c4b9046](https://github.com/user-attachments/assets/0fcd1989-25b6-49f3-b62e-bedfa2aecf8f)

![image](https://github.com/user-attachments/assets/f2e94e6f-146a-448d-84de-3b88abe6edd1)

**Comment by c-o-n-n-y** [comment]
Hi

Traditional orbit prediction methods, such as those based on TLEs, have limitations in accuracy, especially over long periods of time and in the dynamic environment of space.
Depending on the object, you may have deviations of 1 km or more. This is what you see in Figure 2. You have compared the orbit based on TLEs with a precise orbit. So, deviations of several hundred meters is typical due to the known inaccuracy of TLE data.

For your results in Figure 1, you have compared two tools which propagate orbits, both based on TLEs.
The orbits calculated with STK or GROOPS are not exactly the same, they differ from each other.

Please make sure that you are using the same reference and time system when comparing orbit data.

Best regards,
Conny

**Reply by zdwangjian5** [reply]
Thank you for your kindness!
